### PR TITLE
fix(ui): fix textarea overlapping over label on scroll

### PR DESCRIPTION
**Pull request by wa0x6e**
### Summary

Closes: #369

### How to test

1. Go to http://localhost:8080/#/create
2. Fill in the textarea with long info
3. Scrolling the textarea content should not overlap with the label anymore

### CODE REVIEW

File: apps/ui/src/components/Ui/Textarea.vue [hunk 1]
```diff
@@ -37,7 +37,7 @@ watch(model, () => {
   <UiWrapperInput :definition="definition" :error="error" :dirty="dirty">
     <textarea
       v-model="inputValue"
-      class="s-input"
+      class="s-input !py-0 !border-t-[30px] !border-b-[7px] border-transparent"
```
Comment (Sekhmet) [inline apps/ui/src/components/Ui/Textarea.vue:40]:
```suggestion
      class="s-input !py-0 !border-t-[30px] border-transparent"
```

I think it looks better without padding at bottom, but maybe we should ask Fab. With current styles we get some text cut off.

image

Comment (Sekhmet) [reply apps/ui/src/components/Ui/Textarea.vue:40]: @bonustrack

Comment (bonustrack) [reply apps/ui/src/components/Ui/Textarea.vue:40]: Agree, also would be better to have a gradient on top, similar than the sidebar
image

**Review by bonustrack — APPROVED** [review]
tACK

**Review by Sekhmet — APPROVED** [review]
tACK

**Pull request merged**